### PR TITLE
[JENKINS-62918] Table to divs compatibility

### PR DESCRIPTION
**Pull request by alecharp**
Updating the parent pom was necessary for the test and I kept it in the changeset as it cannot hurt to have it (and the BOM version) in recent version.

cc @timja

### CODE REVIEW

File: src/main/resources/lib/auth/blockWrapper.jelly
Comment (timja) [inline src/main/resources/lib/auth/blockWrapper.jelly:1]: you're missing the empty marker file 'taglib'

Comment (alecharp) [reply src/main/resources/lib/auth/blockWrapper.jelly:1]: It's already there on the repository. see https://github.com/jenkinsci/google-oauth-plugin/tree/develop/src/main/resources/lib/auth

Comment (timja) [reply src/main/resources/lib/auth/blockWrapper.jelly:1]: ah sure, I'm used to people missing that and that being the reason why the build has failed

File: pom.xml
Comment (timja) [inline pom.xml]:
```suggestion
        <artifactId>bom-2.222.x</artifactId>
```

File: pom.xml
Comment (timja) [inline pom.xml]: 2.138 hasn't been supported for a long time, 2.222.x is the last supported version currently although it will get dropped in ~march probably
```suggestion
        <version>20</version>
```

File: pom.xml
Comment (alecharp) [reply pom.xml]: At first, I didn't want to bump the minimum core version require for the plugin but I agree it would be nice.

**Comment by MRamonLeon** [comment]
We need to change the Jenkinsfile to run in the required Jenkins core versions. Failure:
```
[ERROR] Failed to execute goal org.jenkins-ci.tools:maven-hpi-plugin:3.17:validate-hpi (default-validate-hpi) on project google-oauth-plugin: Dependency org.jenkins-ci.plugins:credentials:jar:2.3.14 requires Jenkins 2.222.4 or higher. -> [Help 1]
[2021-01-04T16:04:01.226Z] org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.jenkins-ci.tools:maven-hpi-plugin:3.17:validate-hpi (default-validate-hpi) on project google-oauth-plugin: Dependency org.jenkins-ci.plugins:credentials:jar:2.3.14 requires Jenkins 2.222.4 or higher.
```

**Comment by alecharp** [comment]
it won't help here, as I'm not an admin nor a maintainer, the `Jenkinsfile` I would submit wouldn't be used in building the PR..

**Comment by timja** [comment]
> it won't help here, as I'm not an admin nor a maintainer, the `Jenkinsfile` I would submit wouldn't be used in building the PR..

normally I submit a separate PR to fix that then rebuild this PR after merge

**Review by MRamonLeon — APPROVED** [review]
The code despite the Ci failures seems ok to me

**Comment by alecharp** [comment]
BTW @timja, I opened PR #100 to update the Jenkinsfile.

**Comment by MRamonLeon** [comment]
Don't we need to add the version to the Jenkinsfile? To bump the version over 2.222.4
```
buildPlugin(configurations: [
    [ platform: "linux", jdk: "8", jenkins: "2.222.4" ],
    [ platform: "linux", jdk: "11", jenkins: "2.222.4", javaLevel: "8" ],
    [ platform: "windows", jdk: "8", jenkins: "2.222.4" ],
])
```

**Comment by alecharp** [comment]
@MRamonLeon the Jenkinsfile was changed in #100. I just merged the main branch in here, as #100 was merged.

File: pom.xml
Comment (timja) [inline pom.xml]: I suspect this won't work, some plugins in the bom don't use the .x version,

```suggestion
    <jenkins.version>2.222.4</jenkins.version>
```

File: pom.xml
Comment (timja) [inline pom.xml]:
```suggestion
        <version>23</version>
```

**Pull request merged**
